sycl/plugins/unified_runtime/CMakeLists.txt

```diff
@@ -94,14 +94,8 @@ if(SYCL_PI_UR_USE_FETCH_CONTENT)
       CACHE PATH "Path to external '${name}' adapter source dir" FORCE)
   endfunction()
 
-  set(UNIFIED_RUNTIME_REPO "https://github.com/oneapi-src/unified-runtime.git")
-  # commit 1906ce76c9c2d2112584cc0679c25f00c4935e73
-  # Merge: 3e3d8748 7f9b30f6
-  # Author: Kenneth Benzie (Benie) <k.benzie@codeplay.com>
-  # Date:   Tue Apr 30 11:25:19 2024 +0100
-  #     Merge pull request #1452 from jinge90/omp_device_global_intercept
-  #     [ASAN] Intercept urProgramLink in sanitizer layer
-  set(UNIFIED_RUNTIME_TAG 1906ce76c9c2d2112584cc0679c25f00c4935e73)
+  set(UNIFIED_RUNTIME_REPO "https://github.com/lbushi25/unified-runtime.git")
```

Comment:
These changes should be reverted before we merge the PR, or this URL & hash should be updated to point to the main UR branch in the main repo.

I don't think that this PR has a strict dependency on the UR. Since the API you are implementing is a hint, completely ignoring it technically a valid implementation. The fact that `urQueueFlush` does nothing for L0 backend is a UR bug, not SYCL RT bug